l10n_ir_country_states/README.rst

```diff
@@ -0,0 +1,42 @@
+Iran Country States
```

Comment:
Well, that's not mandatory indeed. It's only an advise (it won't make go red the Travis status). It's intended for having a nice banner in Odoo Apps.